src/.env.sample

```
@@ -0,0 +1,259 @@
+
+# Hostname: If the service runs in Docker, the host is the self creation portal; if not, it is localhost.
+APPLICATION_HOST=http://localhost:6001
+
+#Service environment
+APPLICATION_ENV = development
+
+#Route after base url
+APPLICATION_BASE_URL = /scp/
+
+#Token secret to verify the access token
+ACCESS_TOKEN_SECRET = 'bsj82AHBxahusub12yexlashsbxAXADHBlaj'
```

dotenv-linter (3.3.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)